assets/css/style.css

```diff
@@ -3467,8 +3467,3 @@ body.dark-mode .h3 {
   .fab.fa-twitter{
     font-family:sans-serif;
   }
-  .fab.fa-twitter::before{
-    content:"𝕏";
```

Comment:
This part was deleted by mistake as a part of learning. in next commit, i have added this back.